### PR TITLE
data_manager_fetch_ncbi_taxonomy: Provide default value for NCBI taxonomy URL

### PR DESCRIPTION
**Pull request by pvanheus**
FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [x] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [x] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

### CODE REVIEW

**Comment by pvanheus** [comment]
Besides the default value mentioned above, this also bumps the version number so that the change ends up in the toolshed. The "linting" fix applied by @mvdbeek in bf7cd561f3f4c7dbe030dfd2cb010a4e021ed96c was very helpful in fixing a Python 2 vs Python 3 issue but unfortunately has not ended up in the toolshed (with the result that the tool fails on my Python 3 Galaxy).

File: data_managers/data_manager_fetch_ncbi_taxonomy/data_manager/ncbi_taxonomy_fetcher.xml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0"?>
+<?xml version="1.1"?>
```
Comment (bgruening) [inline data_managers/data_manager_fetch_ncbi_taxonomy/data_manager/ncbi_taxonomy_fetcher.xml:1]: this looks wrong :)

Comment (pvanheus) [reply data_managers/data_manager_fetch_ncbi_taxonomy/data_manager/ncbi_taxonomy_fetcher.xml:1]: oops!

**Comment by peterjc** [comment]
Nice - cross reference https://github.com/peterjc/galaxy_blast/issues/111

**Pull request merged**